src/components/data-hooks.js

```diff
@@ -81,13 +116,25 @@ export function useCellsData(loaders, dataset, setItemIsReady, addUrl, isRequire
     if (loaders[dataset].loaders.cells) {
       loaders[dataset].loaders.cells.load().catch(e => warn(e, setWarning)).then((payload) => {
         if (!payload) return;
-        const { data, url } = payload;
+        const { data, url, coordinationValues } = payload;
         setCells(data);
         setCellsCount(Object.keys(data).length);
         addUrl(url, 'Cells');
-        if (onLoad) {
-          onLoad();
-        }
+        // This dataset has cells, so set up the
+        // spatial cells layer coordination value
+        // using the cell layer singleton.
+        const mergedCoordinationValues = {
+          spatialCellsLayers: [
+            {
+              opacity: 1, radius: 50, visible: true, stroked: false,
+            },
+          ],
+          ...coordinationValues,
+        };
```

Comment:
What was this for (and the similar line in the molecules hook)?

Reply:
This was for the automatic initialization of the cells layer and molecules layer if those are present. Similar idea to the auto image layer initialization. Right now none of these parameters (opacity, radius, etc.) would ever be defined in the JSON file (or AnnData Zarr store), but I could imagine in the future they might be.

Reply:
So could we change this to 
```
const coordinationValuesOrDefault = coordinationValues || {
          spatialMoleculesLayers: [DEFAULT_MOLECULES_LAYER],
        };
```
and if the loader does send the `coordinationValues` then just use that.  The way it is now, I think the `coordinationValues` are basically ignored (like in the `wang-2019` example)

Reply:
Does that work?

Reply:
I think

```js
        const mergedCoordinationValues = {
          spatialMoleculesLayers: DEFAULT_MOLECULES_LAYER,
          ...coordinationValues,
        };
```

is better because both

- it allows the loader to set other coordination values besides `spatialMoleculesLayers` while also using the `DEFAULT_MOLECULES_LAYER` if a value for `spatialMoleculesLayers` is not included
- if the loader sets a custom value for `spatialMoleculesLayers` then it will be prioritized over the `DEFAULT_MOLECULES_LAYER` value (since the spread `...coordinationValues` comes after the default).